src/main/resources/templates/orderslist.html

```diff
@@ -0,0 +1,156 @@
+<html xmlns:th=”http://www.thymeleaf.org”>
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>your orders</title>
+
+    <!-- Custom fonts for this template-->
+    <link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i"
+          rel="stylesheet">
+
+    <!--     Custom styles for this template-->
+    <link href="css/sb-admin-2.min.css" rel="stylesheet">
+
+</head>
+
+<body id="page-top">
+
+
+<!-- Page Wrapper -->
+<div id="wrapper">
+
+    <!-- Sidebar -->
+    <ul th:replace="fragments/sidebar :: sidebar"></ul>
+    <!-- End of Sidebar -->
+
+    <!-- Content Wrapper -->
+    <div id="content-wrapper" class="d-flex flex-column">
+
+        <!-- Main Content -->
+        <div id="content">
+
+            <!-- Topbar -->
+            <!--         Footer -->
+            <nav th:replace="fragments/navbar :: navbar"></nav>
+            <!--         End of Footer -->
+            <!-- End of Topbar -->
+
+            <!-- Begin Page Content -->
+            <div class="container-fluid">
+
+                <!-- DataTales Example -->
+                <div class="card shadow mb-4">
+                    <div class="card-header py-3">
+                        <h6 class="m-0 font-weight-bold text-primary">Turbochargers</h6>
+                        <hr>
+                        <a href="addTurbo" class="btn btn-secondary btn-icon-split">
+                    <span class="icon text-white-50">
+                      <i class="fas fa-arrow-right"></i>
+                    </span>
+                            <span class="text">Create</span>
+                        </a>
+                    </div>
+                    <div class="card-body">
+                        <div class="table-responsive">
+                            <table class="table table-bordered" id="dataTable" width="100%" cellspacing="0">
+                                <thead>
+                                <tr>
+                                    <th>Id</th>
+                                    <th>Number</th>
+<!--                                    <th>Model</th>-->
+<!--                                    <th>Parts</th>-->
+<!--                                    <th>Reason</th>-->
+                                </tr>
+                                </thead>
+                                <tbody>
+                                <tr th:each="order : ${listorder}">
+                                    <td th:text="${order.id}"></td>
+                                    <td th:each="turbo : ${order.turbocharger}"
+                                        th:text="${turbo.number}"></td>
```

Comment:
listorder to lista orderów - `List<Order>`
każdy `Order` ma pole `Turbocharger turbocharger`, ale to pole nie jest kolekcją, a Ty próbujesz robić iterację `<td th:each="turbo : ${order.turbocharger}"`. 

Zamiast:
```
<td th:each="turbo : ${order.turbocharger}"
       th:text="${turbo.number}"></td>
```
powinno być:
`<td th:text="${order.turbocharger.number}"></td>`

Reply:
Próbowaliśmy tego rozwiązania, niestety nie działa. Mamy jeszcze inne możliwości?

Reply:
![error](https://user-images.githubusercontent.com/55912906/80512918-8e081380-897e-11ea-8837-69dc91c7b65d.PNG)

Reply:
Kurcze... nie wiem co mam wam napisać... u mnie działa :)
Mam coś takiego

```
                                    <td th:text="${order.turbocharger.number}"></td>
<!--                                    <td th:each="turbo : ${order.turbocharger}"-->
<!--                                        th:text="${turbo.number}"></td>-->
```

i żadnego błędu nie mam. Poszukałem trochę po stackoverflow i ludzie zgłaszali takie problemy w 2016 roku w jakieś wersji thymeleafa, ale ja nie modyfikowałem poma, więc mamy tę samą wersję i u mnie działa. Spróbujcie może dodać dependency do thymeleaf

```
<dependency>
            <groupId>org.thymeleaf</groupId>
            <artifactId>thymeleaf</artifactId>
            <version>3.0.11.RELEASE</version>
        </dependency>
        <dependency>
            <groupId>org.thymeleaf</groupId>
            <artifactId>thymeleaf-spring5</artifactId>
            <version>3.0.11.RELEASE</version>
        </dependency>
```

Reply:
U nas niestety nie działa. Dodaliśmy dependency w pom.xml i nadal pojawia się ten sam błąd. Czy wersja sdk może mieć na to jakiś wpływ? Nie mamy pomysłu na rozwiązanie tego problemu bez tego nie pójdziemy dalej.

Reply:
Błąd, o którym piszesz macie na branchu features/orderlist?